core/trino-main/src/main/java/io/trino/metadata/MetadataManager.java

```diff
@@ -605,7 +608,19 @@ public List<TableColumnsMetadata> listTableColumns(Session session, QualifiedTab
                             }
                         }
                         catch (RuntimeException e) {
-                            if (!(e instanceof TrinoException trinoException) || !trinoException.getErrorCode().equals(UNSUPPORTED_TABLE_TYPE.toErrorCode())) {
+                            boolean silent = false;
+                            if (e instanceof TrinoException trinoException) {
+                                ErrorCode errorCode = trinoException.getErrorCode();
+                                silent = errorCode.equals(UNSUPPORTED_TABLE_TYPE.toErrorCode()) ||
```

Comment:
nit: I like operators to be moved to the next line. Like `.` when methods are chained. It is clearer what is the relationship between operands that way.

Reply:
i like it too, but in many places we use end-of-line. like in equals impls.
the advantage is that every line ends with `||` or ';`

will leave as is for now